plugin-tooling.mk

```diff
@@ -45,11 +46,14 @@ PUBLISHER ?= tzcli
 VENDOR ?= vmware
 PLUGIN_PUBLISH_REPOSITORY ?= localhost:$(REGISTRY_PORT)/test/v1/tanzu-cli/plugins
 PLUGIN_INVENTORY_IMAGE_TAG ?= latest
-OVERRIDE_INVENTORY_IMAGE ?=
+
+PLUGIN_SCOPE_ASSOCIATION_FILE ?= ./cmd/plugin/plugin-scope-association.yaml
+PLUGIN_GROUP_NAME ?= 
 
 # Process configuration and setup additional variables
+OVERRIDE ?=
 OVERRIDE_FLAG = 
-ifneq ($(strip $(OVERRIDE_INVENTORY_IMAGE)),)
+ifneq ($(strip $(OVERRIDE)),)
```

Comment:
Could this collide with a variable in the users environment?
Should prefix those variables with `TANZU_CLI` or `TANZU_BUILDER`?